vars/ciMetrics.groovy

```diff
@@ -0,0 +1,51 @@
+import org.centos.pipeline.CIMetrics
+
+/*
+A class to store build metrics over the lifetime of the build.
+Metrics are stored in customDataMap and then sent to influx at
+the end of the job. Example usage:
+
+try {
+    def stepName = "mystep"
+    ciMetrics.timed stepName, {
+        stage(stepName) {
+            echo "in mystep"
+        }
+    }
+    currentBuild.result = "SUCCESS"
+} catch(err) {
+    currentBuild.result = "FAILED"
+    throw err
+} finally {
+    ciMetrics.writeToInflux()
+}
+ */
+class ciMetrics {
+
+    // A map to store the data sent to influx
+    def customDataMap = ["ci_pipeline": [:]]
+    // This will prefix the data sent to influx. Usually set to the job name.
+    def prefix = "ci_pipeline"
```

Comment:
I'd perhaps set this to default to the job name, and allow overiding?

Reply:
Good suggestion. I will update this.

Reply:
I'm going to leave the defaults as is for now. Maybe just an update to the docs. Since this will prefix the job name.